### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by ridhimamalhotra**
Hi, I implemented feature1. Please check and verify it.
I made following changes:-
1) Created one file.
2) Edit one file.

### CODE REVIEW

File: pull-request-in-action/new-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi i am new file created in feature1 branch.
```
Comment (NidhiBuddhiraja) [inline pull-request-in-action/new-file.txt:1]: happy to see these lines of code here.

Comment (ridhimamalhotra) [reply pull-request-in-action/new-file.txt:1]: Thanks a lot.

**Review by NidhiBuddhiraja — CHANGES_REQUESTED** [review]
Edit file file-second.txt.

**Review by NidhiBuddhiraja — APPROVED** [review]
Great Job as expected from you.

**Pull request merged**